core/server/master/src/main/java/alluxio/master/AlluxioSecondaryMaster.java

```diff
@@ -91,8 +89,7 @@ public static void main(String[] args) {
    * @throws IOException if any I/O errors occur
    */
   private void connectToUFS() throws IOException {
-    String ufsAddress = Configuration.get(PropertyKey.UNDERFS_ADDRESS);
-    UnderFileSystem ufs = UnderFileSystem.Factory.get(ufsAddress);
+    UnderFileSystem ufs = UnderFileSystem.Factory.getForRoot();
```

Comment:
FWIW, `connectToUFS` with `MASTER_KEYTAB_KEY_FILE` and `WORKER_KEYTAB_KEY_FILE` not actually used by typical Alluxio deployment with secure-HDFS. We can reply on the ticket cache (pre-existing `kinit`), without such keytab files, to connect with secure-HDFS.